.github/ISSUE_TEMPLATE/feature_request.md

```diff
@@ -0,0 +1,19 @@
+---
+name: Feature request
+about: Suggest an idea for this project
+title: ''
+labels: ''
```

Comment:
Similar to the bug report template, do you think we should add the `enhancement` label as the default one?